tests/factories.py

```diff
@@ -1,11 +1,13 @@
+from datetime import date
+
 import factory
 
 from .models import EncryptedModel
 
 
 class EncryptedModelFactory(factory.DjangoModelFactory):
     """Factory to generate hashed and encrypted data."""
-    class Meta:
+    class Meta:  # NOQA
```

Comment:
Consider it done.... There must of been a change of behavior in coverage since that requirement was upgraded yesterday. The `#NOQA` was needed in the past otherwise it complained about those lines.